.gitignore

```diff
@@ -15,5 +15,7 @@ javadoc/
 *.war
 *.ear
 
+*my
```

Comment:
I'm not sure if we'll want to merge this or not...  I'll ask Conrad and the TAs about the convenience links.